compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -696,19 +642,6 @@ OMR::SymbolReferenceTable::createKnownStaticDataSymbolRef(void *dataAddress, TR:
    return TR::SymbolReference::create(self(), sym, knownObjectIndex);
    }
 
-TR::SymbolReference *
-OMR::SymbolReferenceTable::createKnownStaticReferenceSymbolRef(void *dataAddress, TR::KnownObjectTable::Index knownObjectIndex)
```

Comment:
This is not Java specific and is part of more general infrastructure that will be useful to other languages - this should not be moved down.